src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -47,17 +48,25 @@ const SubsidiesSummary = ({ className, showSearchCoursesCta }) => {
     request => request.state === SUBSIDY_REQUEST_STATE.REQUESTED,
   ), [couponCodeRequests.length]);
 
-  if (!(hasActiveSubsidies || pendingCouponCodeRequests.length)) {
+  const pendingLicenseRequest = useMemo(() => licenseRequests.find(
+    request => request.state === SUBSIDY_REQUEST_STATE.PENDING_REVIEW,
+  ), [licenseRequests.length]);
+
+  const hasSubsidyRequests = pendingCouponCodeRequests.length > 0 || !!pendingLicenseRequest;
+
+  if (!(hasActiveSubsidies || hasSubsidyRequests)) {
     return null;
   }
 
   return (
     <SidebarCard cardClassNames="border-primary border-brand-primary catalog-access-card mb-5">
       <div className={className} data-testid="subsidies-summary">
         {
-          (subscriptionPlan && userSubscriptionLicense?.status === LICENSE_STATUS.ACTIVATED) && (
+          ((subscriptionPlan
```

Comment:
tiny/optional nit: maybe abstract this conditional logic into a variable above and reference the singular variable here? might improve the readability a bit.